### ISSUE TITLE
Copy the content of Pipeline section in AML_debug.ipynb to AML.ipynb and make sure it works the same

### ISSUE THREAD

**Issue by ariansajina** [alexander34ro/AML-Replicating-Novel-Deep-Learning-with-CNN-and-Bi-Directional-LSTM-for-Improved-Index-Prediction#12]

**Comment by amir-souri** [comment]
I updated the AML.ipynb based on what we did today. I also refined it and add some helper functions to make it clean and readable. Moreover, I expanded it to three pipelines, and it works and learns. Updates to 1016788290 from 37897783314 after 500 iterations.

However,  there is a warning that says, "WARNING:tensorflow:Gradients do not exist for variables ['Variable:0', 'Variable:0', 'Variable:0', 'Variable:0', 'Variable:0', 'Variable:0'] when minimizing the loss". 
It may be due to the vanishing gradient problem and by adding the LSTM layers it disappears!
